scripts/local-setup.sh

```diff
@@ -20,7 +20,7 @@ ensure_value SLACK_APP_USERNAME bennett_test_bot "$ENV_FILE"
 ensure_value GCP_CREDENTIALS_PATH "$GCP_CREDENTIALS_PATH" "$ENV_FILE"
 
 # shellcheck disable=SC1091
-. .env
+. $ENV_FILE
```

Comment:
I think we will want to wrap this in double quotes, but also, we should include [shellcheck](https://github.com/koalaman/shellcheck) in `just check`. I'll leave it as is now and let shellcheck tell us if it needs to be updated.